CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - `PUT /api/install-agent-plugin`. #3417
 - `GET /api/agent-plugins/installed/manifests`. #3424
 - `GET /api/agent-plugins/available/index`. #3420
+- `POST /api/uninstall-agent-plugin/<string:plugin_type>/<string:name>` # 3422
```

Comment:
If this is an RPC endpoint, the body should accept a json document consisting of the type and name.